src/app/styles/globals.css

```
@@ -172,6 +180,65 @@
+}
+
+@layer base {
+  * {
+    @apply border-border;
+  }
+  body {
+    @apply bg-background text-foreground;
+  }
+
+  /* Respect user motion preferences */
+  /*@media (prefers-reduced-motion: reduce) {
```

Stylelint (17.10.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)